### DISCUSSION TITLE
configure DB instance size with environment variable

### DISCUSSION THREAD

**Discussion by hrodmn** [developmentseed/eoAPI#129] [Ideas]
Related to discussion about configuring the database for higher concurrency in #121, it would be nice to be able to set the size of the burstable db instance from an environment variable rather than always defaulting to t3.small.

e.g. `CDK_EOAPI_DB_SIZE=large`

https://github.com/developmentseed/eoAPI/blob/main/infrastructure/aws/cdk/app.py#L162-L164

**Comment by hrodmn** [comment]
I see that this feature is already implemented in eoapi-template + eoapi-cdk!

https://github.com/developmentseed/eoapi-template/blob/main/config.py#L44-L46

https://github.com/developmentseed/eoapi-template/blob/main/eoapi_template/pgStacInfra.py#L37-L51

I'm excited to see how this repo starts to incorporate the infrastructure from eoapi-cdk.

**Reply by vincentsarago** [reply]
🙏 yeah I hope we can switch to eoapi-cdk soon 

while we wait for the update, maybe we can add the option in the actual CDK app, I'll be more than happy to review a PR 🙏
